lib/SPIRV/libSPIRV/spirv.hpp

```diff
@@ -71,21 +71,18 @@ enum ExecutionModel {
     ExecutionModelFragment = 4,
     ExecutionModelGLCompute = 5,
     ExecutionModelKernel = 6,
-    ExecutionModelCount, /* internal use only */
```

Comment:
What you could do is define your own internal header file with your own Count definitions, and then write a test that will fail once the SPV_VERSION from spirv.h is updated.  That forces you to update the Counts if they have changed.